src/app.rs

```diff
@@ -32,6 +34,11 @@ pub mod logging;
 pub mod patch_renderer;
 pub mod screens;
 
+pub enum PatchFound {
+    Found,
+    NotFound
+} 
```

Comment:
I believe this naming is a bit unidiomatic. 
If you look at the usage, `PatchFound::Found` and `PatchFound::NotFound` is a bit "funny".

I would like to propose a change that goes a bit beyond just changing the name.
My suggestion is to name it `B4Result`, and the `lore_session::download_patchset` function should return it.
Doing this, the "file exists" check (see image) could also be moved inside the `download_patchset` function too, and the enum can carry the Path for us.
![image](https://github.com/user-attachments/assets/bf937e1b-f12b-477b-b160-6cd6b75e159a)

```suggestion
pub enum B4Result {
    PatchFound(PathBuf),
    PatchNotFound,
}
```